core/src/migrations/000084-optionsProfileToRecords.ts

```diff
@@ -0,0 +1,45 @@
+import Sequelize from "sequelize";
+
+export default {
+  up: async (queryInterface: Sequelize.QueryInterface) => {
+    await queryInterface.sequelize.query(
+      `UPDATE "config" SET "key"='membershipRecordField' WHERE "key"='membershipProfileField'`
```

Comment:
... we don't have a `config` table?

Reply:
Perhaps this should be about updating the `options` table?   Also, you'll likely need to do a SELECT query first to find the destinations that are of the type `salesforce-object-export` (I *think* that's the same), then update the options that are owned by those destinations